sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -30,24 +30,22 @@
 
 
 class EventHubConsumerClient(ClientBase):
-    """The EventHubProducerClient class defines a high level interface for
+    """The EventHubConsumerClient class defines a high level interface for
     receiving events from the Azure Event Hubs service.
 
     The main goal of `EventHubConsumerClient` is to receive events from all partitions of an EventHub with
     load-balancing and checkpointing.
 
     When multiple `EventHubConsumerClient` operate within one or more processes or machines targeting the same
     checkpointing location, they will balance automatically.
-    To enable the load-balancing and / or checkpointing, checkpoint_store must be set when creating the
-    `EventHubConsumerClient`.
+    To enable load-balancing and checkpoint, checkpoint_store must be set when creating the `EventHubConsumerClient`.
```

Comment:
"checkpoints" I think, or "persisted checkpoints"

Reply:
updated, persisted checkpoint is good description

Reply:
updated, persisted checkpoint is good description